assets/img/homepage/DPC_Logo.svg

```diff
@@ -0,0 +1,122 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This file is named `DPC_Logo`.  Should it be changed to `BCDA_Logo`?

Reply:
Yikes, yes, that would be wise.

Reply:
This is the logo for DPC. There also is a `BCDA_Logo` but since it is used multiple places I set it up you just have to call `{{ site.logourl | relative_url }}` whenever you want to use it